text/0000-dynamic-events.md

```diff
@@ -0,0 +1,146 @@
+---
+title: `lwc:on` directive 
+status: DRAFTED
+created_at: 2025-01-13
+updated_at: 2025-01-13
+pr: (leave this empty until the PR is created)
+---
+
+# `lwc:on` Directive
+
+## Summary
+
+This proposal adds a mechanism to add a collection of event listeners to elements in an LWC template using a new directive `lwc:on`.
+
+## Basic example
+
+```js
+// x/myComponent.js
+export default class MyComponent extends LightningElement {
+
+    childEventHandlers = {
+        foo: function (event) {console.log('foo');} ,
+        bar: function (event) {console.log('bar');}

```

Comment:
It would be great if one of these examples used `this`, to demonstrate that the listeners are bound to the component instance.